builder/dockerfile/dispatchers_test.go

```diff
@@ -256,27 +256,6 @@ func TestOnbuild(t *testing.T) {
 	}
 }
 
-func TestWorkdir(t *testing.T) {
```

Comment:
In the previous commit, I skipped the test on Windows as the new path sent us calling through the backend into the daemon which wasn't setup in unit tests. Of course, with the latest fix described above, now that we only callback if the disableCommit flag isn't set, the test can be fully re-instated. So it'll be back in for the next update coming shortly.